src/client/testing/page-objects/view-availability-appointment-pages/month-view-availability-page.ts

```diff
@@ -1,17 +1,122 @@
 import { type Locator, type Page, expect } from '@playwright/test';
 import RootPage from '../root';
 
-export default class ViewAvailabilityPage extends RootPage {
+export type weekOverview = {
+  header: string;
+  services: serviceOverview[];
+  totalAppointments: number;
+  booked: number;
+  unbooked: number;
+};
+
+export type serviceOverview = {
+  serviceName: string;
+  bookedAppointments: number;
+};
```

Comment:
Minor thing but elsewhere we're using PascalCase for types

Also I may have missed it, but are these used anywhere else? Might not need to be exported?